aiogram/dispatcher/fsm/state.py

```diff
@@ -54,6 +54,13 @@ def __call__(self, event: TelegramObject, raw_state: Optional[str] = None) -> bo
             return True
         return raw_state == self.state
 
+    def __copy__(self) -> "State":
+        return self
```

Comment:
`__copy__` method should return a new instance of this class instead of the same instance, but after copying the old instance should be equals with new instance

Reply:
I wanted to implement the same behavior for states as for enums, since they are used in a very similar way.

I thought that having several instances of a state when copying it (for example, a pydantic copies objects by default during validation) can lead to hard-to-detect errors.

And changing the copy behavior at the framework level can reduce the number of such errors from users


```python3
>>> from enum import Enum
>>> from copy import copy
>>> class Bar:
...     ...
... 
>>> class Foo:
...     bar = Bar()
... 
>>> print(Foo.bar is copy(Foo.bar))
False
>>> class FooEnum(Enum):
...     bar = Bar()
... 
>>> print(FooEnum.bar is copy(FooEnum.bar))
True
>>> 

```